index.html

```diff
@@ -6190,6 +6200,9 @@ <h2>Changes since JSON-LD Community Group Final Report</h2>
       This allows a mechanism for documenting the content of a context using HTML.</li>
     <li>Consolidate <a>RemoteDocument</a> processing into the <a>LoadDocumentCallback</a>
       including variations on HTML processing.</li>
+    <li>The <a href="#iri-compaction">IRI compaction algorithm</a> may generate an error if the result is an
+      <a>absolute IRI</a> where the scheme part could be confused with a term in the
```

Comment:
```suggestion
      <a>absolute IRI</a> which could be confused with a compact IRI in the
```
* since the condition is actually a little more complex